sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -16,6 +16,15 @@
   - `UsernamePasswordCredential`
   - `VisualStudioCodeCredential`
 
+### Breaking Changes
+
+- Credential types supporting multi-tenant authentication will now throw `ClientAuthenticationError` if the requested tenant ID doesn't match the credential's tenant ID, and is not included in `additionally_allowed_tenants`. Applications must now explicitly add additional tenants to the `additionally_allowed_tenants` list, or add '*' to list, to enable acquiring tokens from tenants other than the originally specified tenant ID.
+- These beta features in 1.11.0b3 have been removed from this release and will be added back in 1.12.0b1
+  - `tenant_id` for `AzureCliCredential`
+  - removed `VisualStudioCodeCredential` from `DefaultAzureCredential` token chain
+  - `AZURE_CLIENT_CERTIFICATE_PASSWORD` support for `EnvironmentCredential`
+  - `validate_authority` support
+
```

Comment:
Given that these features were never in the GA version is this really a breaking change?

Reply:
I think it's good to call this out, so that customers depending on `1.11.0b3` features realize `1.11.0` doesn't include them as it normally would.

Reply:
Fair enough. I worry a bit it's confusing the other way but I don't have strong feelings.